modules/thrift/src/thrift/config.h

```diff
@@ -0,0 +1,463 @@
+/* config.h.  Generated from config.hin by configure.  */
+/* config.hin.  Generated from configure.ac by autoheader.  */
+
+#ifndef CONFIG_H
+#define CONFIG_H
+
+/* Define if the AI_ADDRCONFIG symbol is unavailable */
+/* #undef AI_ADDRCONFIG */
+
+/* Possible value for SIGNED_RIGHT_SHIFT_IS */
+#define ARITHMETIC_RIGHT_SHIFT 1
+
+/* Defines automake version */
+#define AUTOMAKE_VERSION
+
+/* Use *.h extension for parser header file */
+#define BISON_USE_PARSER_H_EXTENSION 1
+
+/* Defines bison version */
+#define BISON_VERSION 3.8.2
```

Comment:
Is bison a required program for using this module too?

Reply:
No. it's only required when building the thrift compiler. So technically this might be irrelevant here and could be removed to avoid confusion.